shared/modules/consensus_module.go

```diff
@@ -11,6 +11,9 @@ type ValidatorMap map[string]Actor // TODO (Drewsky) deprecate Validator map or
 
 type ConsensusModule interface {
 	Module
+	ConfigurableModule
```

Comment:
Question: Are there other `modules` that are not configurable and genesis dependent?

I would like to see if we can move some of these interfaces to `Module`

Reply:
I'd rather keep interfaces separated to be honest.

It's true that the majority of modules are `ConfigurableModule` but only a couple are `GenesisDependentModule`.

I'd like to keep `Module` to the bare minimum.